src/Clide/Extensions/TracingExtensions.cs

```diff
@@ -0,0 +1,191 @@
+﻿namespace Clide
+{
+    using Microsoft.VisualStudio;
+    using Microsoft.VisualStudio.Shell.Interop;
+    using System;
+    using System.ComponentModel;
+    using System.Diagnostics;
+    using System.Windows;
+
+    /// <summary>
+    /// Provides tracing extensions on top of <see cref="ITracer"/>.
+    /// </summary>
+    public static class TracingExtensions
```

Comment:
I wonder why this didn't go into `Interfaces`, since it's likely users will want to use it...